backend/entities/clubs/leadership/controller.go

```diff
@@ -0,0 +1,180 @@
+package leadership
```

Comment:
if we're switching to leadership as a term, we should also remove the term "poc" so lID instead of pocID and etc